### DISCUSSION TITLE
Script the removal of old installations?

### DISCUSSION THREAD

**Discussion by pmetzger** [mikebrady/shairport-sync#1748] [Ideas]
The upgrade instructions for both shairport-sync and nqptp both have some manual file removal, systemd commands, etc. required of Linux users to remove the old version. Could `uninstall.sh` files be added to make this easier and quicker?

**Comment by MSL-DA** [comment]
Look here: https://github.com/mikebrady/shairport-sync/discussions/1566

**Comment by pmetzger** [comment]
@MSL-DA If it's not part of the distribution, people won't know about it and won't find it.

**Reply by MSL-DA** [reply]
I agree 👍🏻
